packages/orbit-components/src/InputSelect/InputSelect.ct-story.tsx

```diff
@@ -68,6 +68,12 @@ export function InputSelectStory() {
         placeholder="Custom Placeholder"
         disabled
       />
+      <InputSelect
+        options={simpleOptions}
+        label="Custom label (readOnly)"
+        defaultSelected={simpleOptions[0]}
```

Comment:
I set `defaultSelected` value instead of a `placeholder`. Now it makes more sense related to `readOnly` prop (not disabled).  🙂